Proposals/xgov-x.md

```diff
@@ -0,0 +1,108 @@
+amount_requested: 1
+status: Draft
+---
+
+## Abstract
+
+The “Transaction Builder UI” (name not final) will be an online tool that provides an easy and user-friendly way to create just about any transaction on Algorand. Creating and sending a transaction on Algorand without an existing tool for that particular type of transaction requires using the SDK or `goal`, which requires programming knowledge and/or a node setup. Just about every wallet makes simple payment transactions easy. However, many of the more advanced transactions are either difficult or impossible within the wallets’ UIs. Most wallets don’t allow most types of Algorand transactions for security and user experience reasons.
+
+## Team
+
+### No-Cash-7970
+
+A lone web developer for over 10 years. In September 2021, she became interested in Algorand and has been a tiny part of the Algorand community ever since as an active member of the Algonauts Official Discord group.
+
+## Experience with Algorand
+
+### No-Cash-7970’s Experience
+
+- Using and experimenting with the JavaScript and Python SDKs
+- Built a working prototype game using Reach
+
+## Roadmap
+
+**Q3 2023** - The Transaction Builder UI will be available to the public to use to create payment, key registration, asset configuration, asset freeze, asset transfer, and application call transactions.
+
+## Benefits for the community
+
+Having an easy-to-use user interface (UI) for creating and sending transactions of almost any level of difficulty would make Algorand an easier blockchain to use. More advanced and complex transactions would be easier and safer for users of various knowledge levels to create and send. An easy-to-use UI would also allow users to obtain a better understanding of what transaction(s) they are sending and how transactions work in general.
+
+## Additional information
+
+### Project Features/Requirements
+
+#### User Interface and User Experience
+
+- Focused. The Transaction Builder UI is solely tool for creating Algorand transactions.
+- Simple. Building an Algorand transaction can get very complex very quickly, so the UI shouldn’t add to that complexity. However, the UI shouldn’t completely hide complex details either. Allow the user to see more of the details if they want to.
+- Show and explain transaction details and the transaction fields, if user desires. The Transaction Builder UI may be used as an education tool.
+- The UI should be easy to use across various devices and screen sizes.
+- Safe defaults. The safest options are set as the default.
+- Allow the user to easily use and switch between MainNet, TestNet, and BetaNet.
+- Allow the user to specify a custom `algod` node.
+
+#### Economics and Finance
+
+- The user must not be charged a fee other than the Algorand network fee(s).
+- The need for a web server should be kept to a minimum. Preferably, the Transaction Builder UI wouldn’t need a back-end web server, so cost of maintaining the Transaction Builder UI is as low as possible.
```

Comment:
Are you thinking everything will pretty much be client side, so just host then in an s3 bucket or something?

Reply:
How about pushing the page to IPFS and minting an ARC-19 NFT that points to the latest version ?
The content will be cached by gateways dedicated to Algorand NFTs so no need to pin it. 

Also this transaction template (ARC19) could be part of the page so others could host such SPAs the WEB3 way as well.